test/distributed/fsdp/test_fsdp_state_dict.py

```diff
@@ -513,6 +555,69 @@ def test_basic_save_and_load_state_dict(
 
             self._compare_models(model, model_new, self.assertEqual, check_fp16=fp16)
 
+    @skip_if_lt_x_gpu(2)
+    @parametrize("state_dict_type", _SUPPORTED_STATE_DICT_IMPLS)
+    @parametrize(
+        "cpu_offload",
+        [CPUOffload(offload_params=True), CPUOffload(offload_params=False)],
+    )
+    @parametrize("mixed_precision", [True, False])
+    @parametrize("state_dict_rank0_and_offload", [True, False])
+    def test_buffers_save_and_load_state_dict(
+        self,
+        state_dict_type: StateDictType,
```

Comment:
The typing is `str` not `StateDictType`. This is a legacy issue that multiple ways to declare `state_dict_type` exist in the same test.

Reply:
good catch! I've now updated for both this test and `test_basic_save_and_load_state_dict`.